pkg/client/config/loader.go

```diff
@@ -0,0 +1,55 @@
+package config
```

Comment:
is this a straight move?

Also, open an issue to discuss removing the auto-migration.  It's been over two years.

Reply:
Not a straight move.
- [NewOpenShiftClientConfigLoadingRules](https://github.com/openshift/origin/blob/master/pkg/oc/cli/config/loader.go#L45) is left duplicated in the original file, as it is needed by packages within the `pkg/oc` subtree
- [NewPathOptions](https://github.com/openshift/origin/blob/master/pkg/oc/cli/config/loader.go#L61) and [NewPathOptionsWithConfig](https://github.com/openshift/origin/blob/master/pkg/oc/cli/config/loader.go#L65) are entirely left in the original file as they are only used by packages within the `pkg/oc` subtree

Reply:
> Not a straight move.
> 
> NewOpenShiftClientConfigLoadingRules is left duplicated in the original file, as it is needed by packages within the pkg/oc subtree
> NewPathOptions and NewPathOptionsWithConfig are entirely left in the original file as they are only used by packages within the pkg/oc subtree

Ok, that's still fine.

Reply:
> Also, open an issue to discuss removing the auto-migration. It's been over two years.

Tracking this here: https://github.com/openshift/origin/issues/17483